engine/src/flutter/impeller/entity/shaders/gradients/conical_gradient_ssbo_fill.frag

```diff
@@ -10,6 +10,8 @@ precision highp float;
 #include <impeller/texture.glsl>
 #include <impeller/types.glsl>
 
+layout(constant_id = 0) const float kind = 3.0;
```

Comment:
This isn't strictly necessary to fix this issue but I did it as part of my experiments and kept it since it should make the conical shader faster.

Reply:
seems like a good idea!

Reply:
dumb question though. we support spec constants via GLSL ifdefs in GLES. If the shader doesnt' crash the parser with IFDEFS instead of separate shaders, it might be a slightly better approach as it reduces the shader count.

On the other hand, 🤷‍♂️ and it might still crash

Reply:
Yea, that was the first thing I tried.  It still crashed so I had to go back and split them up into separate files =T.

Reply:
alas....